atom/src/member.cpp

```diff
@@ -457,17 +454,18 @@ Member_set_index( Member* self, PyObject* value )
 }
 
 
-template<typename T> bool
```

Comment:
what's the reason for changing the return type here?

Reply:
I just thought it was cleaner since the size arg needed added anyways, but either works. I can revert it back if you prefer.